File: proyectos_c#/web/DemoPruebasUnitarias/DemoPruebasUnitarias/Matematicas.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DemoPruebasUnitarias
{
    public class Matematicas
    {
        public static int suma(int sumando1, int sumando2)
        {
            return sumando1 + sumando2;
        }
        public static int division(int dividendo, int divisor)
        {
            if (divisor == 0)
            {
                throw new DivideByZeroException("No se puede dividir entre cero");
            }
            return dividendo / divisor;
        }
    }
}

File: proyectos_basic/web/ProyectoUnitTest/ProyectoUnitTest/Matematicas.vb
Imports Microsoft.Bot.Connector

Public Class Matematicas
    Public Shared Function suma(sumando1 As Integer, sumando2 As Integer) As Integer
        Return sumando1 + sumando2
    End Function

    Public Shared Function division(dividendo As Integer, divisor As Integer) As Integer
        If divisor = 0 Then
            Throw New DivideByZeroException("No se puede dividir entre cero")
        End If
        Return dividendo / divisor
    End Function
End Class
